packages/namekit-react/src/components/ENSTextArea.tsx

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import cc from "classcat";
+
+import { TextArea, TextAreaProps } from "./TextArea";
+
+export const ENSTextarea: React.FC<TextAreaProps> = (props) => {
```

Comment:
@notrab suggest we also rename this to `ENSTextArea` with an uppercase `A` and also update related items in Storybook, ex:

![CleanShot 2024-07-03 at 10 13 37](https://github.com/namehash/nameguard/assets/126201998/01374183-109a-4846-bf0a-84d5fb600e89)

Reply:
Good catch, fixed